docs/docs/training-data-importers.mdx

```diff
@@ -212,3 +212,9 @@ class MyImporter(TrainingDataImporter):
     def _custom_get_nlu_file(self) -> Text:
         pass
 ```
+
+## Migrating your Custom Importer to Rasa 3.0
+
+`TrainingDataImporter` and its subclasses contain no async methods since Rasa 3.0.
+In order to migrate your custom importers and make them work with Rasa 3.0, you also need to
+replace your async methods with synchronised ones.
```

Comment:
@wochinge I honestly don't have any ideas of what else to write here. Should there be an example or is it straightforward enough already?